test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -221,7 +223,7 @@ function RunSaveHelpTests
                 $saveHelpFolder = Join-Path $TestDrive (Get-Random).ToString()
                 New-Item  $saveHelpFolder -Force -ItemType Directory
 
-                It "Validate Save-Help for the '$moduleName' module" {
+                It "Validate Save-Help for the '$moduleName' module" -Pending:$Pending {
```

Comment:
In this test, this single `It` block runs in a loop to test a bunch different modules, so there is no specific test case that you can directly put a `-Pending` at.